File: frontend/apps/admin/features/fa-im-pages/types/group/ImGroupMember.ts
// types/group/ImGroupMember.ts
export interface ImGroupMember {
  id: number;              // 主键ID
  groupId: string;         // 群ID
  userId: number;          // 用户ID
  userNickname?: string;   // 群内昵称
  role?: number;           // 角色: 1成员 2管理员 3群主
  status?: number;         // 状态: 1正常 2退出
  joinTime?: string;       // 加入时间
  crtTime?: string;        // 创建时间
  updTime?: string;        // 更新时间
}

File: fa-im/src/main/java/com/faber/api/im/group/entity/ImGroupMember.java
package com.faber.api.im.group.entity;

import com.faber.core.bean.BaseDelEntity;
import com.baomidou.mybatisplus.annotation.TableName;
import com.baomidou.mybatisplus.annotation.TableField;
import lombok.Data;
import lombok.EqualsAndHashCode;

import java.util.Date;

/**
 * 群成员实体类
 *
 * @author faber
 */
@Data
@EqualsAndHashCode(callSuper = false)
@TableName("im_group_member")
public class ImGroupMember extends BaseDelEntity {
    
    private Long id;
    /**
     * 群ID
     */
    @TableField("group_id")
    private String groupId;

    /**
     * 用户ID
     */
    @TableField("user_id")
    private Long userId;

    /**
     * 群内昵称
     */
    @TableField("user_nickname")
    private String userNickname;

    /**
     * 角色: 1成员 2管理员 3群主
     */
    @TableField("role")
    private Integer role;

    /**
     * 加入时间
     */
    @TableField("join_time")
    private Date joinTime;
}